### PR TITLE
Fix a few typos, mostly in DEVDOC

### PR DESCRIPTION
**Pull request by hevrard**
Bug: n/a

### CODE REVIEW

File: DEVDOC.md [hunk 1]
```diff
@@ -718,7 +718,7 @@ only the (transformed) capture commands needs to be replayed.
 
 If the user-requested replay does require to transform the initial commands,
 then the pre-warm replay is abandonned and a new replay of the transformed
-inital commands and then the transformed capture commands are executed.
+inital commands and then the transformed capture commands is executed.
```
Comment (johnplate) [inline DEVDOC.md:721]: "commands" is plural, so "are" was correct.
The last part of the sentence sounds weird, maybe we should rephrase it to "then the pre-warm replay is abandonned and a new replay of the transformed inital and capture commands are executed."

**Review by johnplate — APPROVED** [review]
LGTM, just one change.

File: DEVDOC.md (hunk 1 above)
Comment (hevrard) [reply DEVDOC.md]: I think the verb conjugates with "replay" here, as in: `a new replay (of <whatever>) is executed.`

Thanks for the suggestion, I applied it. But I kept "is". What do you think?

Comment (johnplate) [reply DEVDOC.md]: Yes, thanks, you are right. I was confused by the initial weird sentence.
